core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.ts

```diff
@@ -65,6 +65,13 @@ angular.module('oppia').directive('feedbackImprovementTask', [
             return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
               $scope.getLatestMessage().updatedOn);
           };
+
+          $scope.wasUpdatedToday = function() {
+            var millisUptillToday = new Date().setHours(0, 0, 0, 0);
+            return (
+              $scope.getLatestMessage().updatedOn - millisUptillToday >= 0 ?
```

Comment:
You don't need the "? true : false" part.

Reply:
Yeah, will do.